appveyor.yml

```diff
@@ -7,7 +7,7 @@ environment:
   access_token:
     secure: ZxcrtxQXwszRYNN6c1ZIagczEqzmQQZeYHY58izcmF0jdq/cptxJvFUoVxDmnoqj
 install:
-  - ps: New-SelfSignedCertificate -DnsName localhost -CertStoreLocation cert:Localmachine\My
+  - ps: Import-Certificate -FilePath test/schannel-ca.crt -CertStoreLocation 'Cert:\CurrentUser\Root' -Verbose
```

Comment:
Hm yeah I was looking into this. Apparently appveyor [has a semi-solution for this](http://www.appveyor.com/docs/deployment/local#installing-self-signed-ssl-certificate-to-a-website) though? I'll try that and see if it works better.